dpdk/dpdk_vrouter.c

```diff
@@ -169,7 +172,8 @@ dpdk_init(void)
     /* init timer subsystem */
     rte_timer_subsystem_init();
 
-    return 0;
+    /* init interface lock */
+    return pthread_mutex_init(&vr_dpdk.if_lock, NULL);
```

Comment:
What is the mutex protecting against? All interface operations (including addition/deletion of monitoring interfaces) happen in the context of the netlink thread inside vrouter. What is the other context - is it the thread handling KNI fifo requests from the kernel? Or something else? A comment with this info would be useful.